src/utils/copy.ts

```diff
@@ -55,7 +57,12 @@ export function shallowCopy(original: any, options?: Options<any, any>) {
     // don't use `Object.create(Object.getPrototypeOf(obj), Object.getOwnPropertyDescriptors(obj));` by default.
     const copy: Record<string | symbol, any> = {};
```

Comment:
It's more concise this way.

```ts
const copy: Record<string | symbol, any> = {};
Reflect.ownKeys(original).forEach((key) => {
  copy![key] = original[key];
});
return copy;
```

Reply:
`Reflex.ownKeys()` also retrieves non-enumerable keys,

<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/ownKeys#description>

but `{ ...object }` does not.

It has to call `Object.prototype.propertyIsEnumerable` in every iteration to filter the keys. The performance improvement over using the spread operator on objects is likely to be minimal.

Reply:
In benchmark, there is no significant performance difference between `Reflect.ownKeys()` and the current implementation. When it comes to the integrity of shallow copy implementation for complex object structures, the rank is `Object.keys` < `Reflect.ownKeys` < `Object.getOwnPropertyDescriptors`. 

Your implementation is indeed consistent with the behavior of spread object operations, but its performance is almost identical to that of `Reflect.ownKeys`. However, `Reflect.ownKeys` has the advantage of ensuring that even non-enumerable keys can be copied.

Reply:
In terms of consistency, there's nothing wrong with your implementation. Thanks for your PR.

Reply:
> However, `Reflect.ownKeys` has the advantage of ensuring that even non-enumerable keys can be copied.

So we do WANT non-enumerable keys in shallow copy? 🤔 If so, then the only difference between `strictCopy` and "normal copy" will be `configurable` and `writable`.

And if we use `Reflect.ownKeys`, this test will fail

https://github.com/unadlib/mutative/blob/6d8600e32030dbaa6c1ddf1deabaccaf790474bc/test/immer/__tests__/not-strict-copy.ts#L25-L40

because in `Object.defineProperty()`, `enumerable` is default to `false`.

<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#enumerable>

Reply:
If we use `Reflect.ownKeys`, it indeed cannot align with Immer's non-strict shallow copy. Considering the alignment with Immer and object spread behavior, we should indeed adopt the implementation in your PR. 

I am adding more test cases and will release v1.0.2 shortly.